packages/ai-jsx/src/next/index.ts

```diff
@@ -0,0 +1,114 @@
+import 'server-only';
+import * as ReactModule from 'react';
+import * as LLMx from '../react/core.js';
+export * from '../react/core.js';
+import { markAsJsxBoundary } from '../react/jsx-boundary.js';
+
+function unwrapReact(partiallyRendered: LLMx.PartiallyRendered): ReactModule.ReactNode {
+  if (LLMx.isElement(partiallyRendered)) {
+    // This should be an AI.React element.
+    if (partiallyRendered.tag !== LLMx.React) {
+      throw new Error('unwrapReact only expects to see AI.React elements or strings.');
+    }
+
+    return partiallyRendered.props.children;
+  }
+
+  return partiallyRendered;
+}
+
+function computeSuffix(prefix: ReactModule.ReactNode[], frame: ReactModule.ReactNode[]) {
+  const computedSuffix = [] as ReactModule.ReactNode[];
+  for (let i = 0; i < frame.length; ++i) {
+    if (i >= prefix.length) {
+      computedSuffix.push(frame[i]);
+    } else if (computedSuffix.length > 0) {
+      return null;
+    } else if (prefix[i] === frame[i]) {
+      continue;
+    } else {
+      const fromPrefix = prefix[i];
+      const fromFrame = frame[i];
+
+      if (typeof fromPrefix === 'string' && typeof fromFrame === 'string' && fromFrame.startsWith(fromPrefix)) {
+        computedSuffix.push(fromFrame.slice(fromPrefix.length));
+      } else {
+        return null;
+      }
+    }
+  }
+
+  return computedSuffix;
+}
+
+/**
+ * A JSX component that allows AI.jsx elements to be used in an RSC component tree.
```

Comment:
```suggestion
 * A JSX component that allows AI.jsx elements to be used in a [NextJS RSC component tree](https://nextjs.org/docs/getting-started/react-essentials#server-components).
```

Reply:
"RSC" and "NextJS RSC" may refer to subtly different things, so as long as we're specifically testing against NextJS, I think we should be clear about that in the language.